### ISSUE TITLE
Using Gaussian Splatting that backpropagates into camera positions 

### ISSUE THREAD

**Issue by skrya** [dcharatan/flowmap#39]
Dear Authors,

Thank you once again for the incredible work you have done!

I have a question regarding the Gaussian splatting technique mentioned in your README (https://github.com/dcharatan/flowmap/blob/main/README.md). Specifically, you mention that this method backpropagates into camera positions.

Could you please clarify whether the extrinsics and intrinsics obtained from Flowmap are robust enough to be used directly with 3D Gaussian Splatting without the need for further optimization of the camera positions while emploing 3D GS? Additionally, could you share any observations on the differences in NVS performance, in terms of PSNR, with and without camera position optimization when using Gaussian splatting?

Thanks!

**Comment by dcharatan** [comment]
In many cases, FlowMap's poses and intrinsics are accurate enough to be used directly for 3DGS. We found that both COLMAP and FlowMap benefit from pose refinement. FlowMap's poses are generally slightly worse, which means it benefits slightly more from pose refinement. However, the overall picture is the same whether or not you use pose refinement.

We plan to update the paper with rendering metrics without pose refinement. Here's a preview of those results. We ran both COLMAP and FlowMap at the resolution FlowMap's optical flow was computed at, then ran the original 3DGS repo at the default resolution (i.e., scaling down to 1.6K automatically where appropriate). You can see that relatively speaking, COLMAP works better for higher quality captures (Replica, which is synthetic/perfect, LLFF, etc.), while FlowMap can work a bit better for blurrier/more distorted captures (e.g., CO3D).

<img width="585" alt="image" src="https://github.com/dcharatan/flowmap/assets/13124225/8a34f5dd-7a59-4c41-9303-189439e7a235">

If you're interested in getting the highest quality renderings, I would always recommend backpropagating into camera poses and using an LPIPS loss. However, you might consider using a different code base for bundle-adjusting 3DGS, since our implementation slows down training quite a bit compared to the original 3DGS repo.
